packages/fx-core/resource/package.nls.json

```diff
@@ -1175,10 +1175,13 @@
   "driver.deploy.error.unknownError": "Unknown error happened: %s",
   "driver.deploy.notice.deployAcceleration": "Deploying to Azure App Service takes a long time. Consider referring to this document to optimize your deployment:",
   "driver.deploy.error.noFileFindInDistributionFolder": "No file is found in distribution folder. Please check if the distribution folder is empty or you have ignored everything.",
-  "driver.deploy.notice.deployDryRunComplete": "Preparations of deployment are complete. ",
+  "driver.deploy.notice.deployDryRunComplete": "Preparations of deployment are complete. You can find the package in the `%s`",
```

Comment:
Preparations of deployment -> Deployment preparations 
complete -> completed

also for the last sentence, if %s is a folder/path or file, can remove the 'the' before %s

Reply:
fixed